README.md

```diff
@@ -2074,6 +2106,22 @@ keys:
 - `timestamp`: Integer. Unix timestamp in milliseconds.
 - `level`: String. `info`, `error` or `warn`.
 
+## Testing
+
+### Run all
+
+```
+cd src
+python<version> -m unittest -v tests/*.py
+```
+
+### Run just the factory tests
```

Comment:
I don't think there is any need to include in the doc how to run individual tests.  If we do that, we would need to list how to run tests for _every_ module, for example `tests.test_pipeline.TestPipeline`, `tests.test_query.TestQuery`, and a bunch more.  Maybe we should just include something like how to run module tests and include one as an example.